### PR TITLE
Larger mobile autocomplete fonts and paddings

### PR DESCRIPTION
**Pull request by joel-jeremy**
Made autocomplete components easier to tap by making the fonts and paddings larger.

Previous:
image

New:
image

### CODE REVIEW

**Comment by trevdor** [comment]
Yes! We’ve needed to improve this. 
Two thoughts:
1. Would you see what you think of it with an even larger font, but regular weight instead of bold? 
2. How about a subtle list item divider?

Apple’s native lists for comparison:
IMG_6976

**Comment by joel-jeremy** [comment]
@trevdor Updated. :) Let me know if you want me to adjust the font sizes.

**Comment by trevdor** [comment]
Definitely easy tap targets now!

Seems like the font-size could decrease just a hair on list items so they're smaller than group headings. That contrast was nice in your original screenshot. (Maybe our font sizes in `styles.ts` aren't quite up to the task...)

On my phone, grasping the entire diff is tricky, so apologies if I've misread--but how about regular list items get weight 400, font-size...um, a rogue 17?!, and vertical passing of 8? (think I see 10 right now, and rows seem just a touch tall imo)

I haven't had a chance to tweak it in my own browser to give a true "here's my favorite sizing"--it just feels a bit chunky on a small phone at the moment. Thanks for hearing me out!

**Comment by joel-jeremy** [comment]
@trevdor Adjusted the fonts and paddings. :)

**Comment by trevdor** [comment]
Trying to get through the rest of the code tonight, but the visual look good to me!

File: packages/desktop-client/e2e/transactions.test.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ test.describe('Transactions', () => {
       await expect(autocomplete).toMatchThemeScreenshots();
 
       // Select the active item
-      await page.getByRole('button', { name: 'Clothing' }).click();
+      await page.getByTestId('Clothing-category-item').click();
```
Comment (trevdor) [inline packages/desktop-client/e2e/transactions.test.js:66]: Personally, I prefer `getByRole` with the user-visible text over grabbing elements by something the user can't see, in order to keep our tests behaving similarly to a user. Did the existing selector fail after the refactor?

As I say, it's a preference, so ship it as you see fit.

**Review by trevdor — DISMISSED** [review]
Thanks! This will help me personally, if nothing else :D 

Left a few questions/comments, but however you resolve those is good with me.

File: packages/desktop-client/e2e/transactions.test.js (hunk 1 above)
Comment (joel-jeremy) [reply packages/desktop-client/e2e/transactions.test.js:66]: For some reason, the test is hanging on this `getByRole` call so I had to change it. I'm happy to revert it back to use `getByRole` if anybody can get it to work again.

**Comment by SalvyTek** [comment]
@joel-jeremy To improve the readability of the selected category name, could you use a color with greater contrast?  Thanks for increasing the font size in this component!

**Comment by SalvyTek** [comment]
! Thank you!  Now it's perfect!

File: packages/desktop-client/e2e/transactions.test.js (hunk 1 above)
Comment (trevdor) [reply packages/desktop-client/e2e/transactions.test.js:66]: Not worth holding it up for (although we're in code freeze at the moment).

**Review by trevdor — APPROVED** [review]

**Pull request merged**